draft-ietf-jsonpath-base.md

```diff
@@ -1075,7 +1078,6 @@ Notes:
 * A member or element value by itself is *falsy* only, if it does not exist. Otherwise it is *truthy*, resulting in its value. To be more specific explicit comparisons are necessary. This existence test — as an exception of the general rule — also works with complex values.
 * Regular expression tests can be applied to `string` values only.
 * Containment tests work with arrays and objects.
-* Explicit boolean type conversion is done by the not operator `neg-op`.
```

Comment:
I think this type conversion statement is no longer needed since `!` now applies only to existence terms, which give boolean values, and parenthesised boolean expressions which again give boolean values.